plone/recipe/codeanalysis/__init__.py

```diff
@@ -198,6 +199,11 @@ def uninstall_pre_commit_hook(self):
 def code_analysis(options):
     start = time()
 
+    # if there is a second argument (first is always the progam itself)
```

Comment:
@tisto I didn't want to make it much more complex, what would be the benefit of argparse?

Adding argparse would mean that we would allow to configure code-analysis *completely/partially* from the command-line and I guess that's not what we do really want.

The other thing that could be interesting would be to run it on more than one folder, i.e.

```
bin/code-analysis src/project1 src/project2 src/projectX
```

Although I'm not sure if that's really interesting at all, but still would not require argparse either.

What would you like to achieve with argparse?